great-idea/css/index.css

```diff
@@ -73,30 +69,31 @@ nav {
 	flex-direction: row;
 	justify-content: space-around;
 	align-items: baseline;
-	background: #f7f7f7af;
+	background: #f7f7f7;
 	position: fixed;
 	top: 0;
 	left:0;
-	padding: 30px;
+	padding: 15px;
 	z-index: 100;
```

Comment:
LOVE the fixed nav!

Reply:
Thanks, just playing around with stuff. :)